src/BasicRum/Diagram/Builder/RenderType/Metrics/PlaneMetricsInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\BasicRum\Diagram\Builder\RenderType\Metrics;
+
+interface PlaneMetricsInterface
+{
+    public function proceed(int $key): void;
+
+    public function getDataForDiagram(): array;
+
+    public function getExtraDiagramParams(): array;
+
+    public function getExtraLayoutParams(): ?array;
```

Comment:
Why do we have **?array** here? I though that we could return empty array or an array with values.

Reply:
Fixed